app/components/sidebar/sidebar.html

```diff
@@ -9,75 +9,75 @@
   </div>
   <div class="sidebar-content">
   <ul class="sidebar">
-    <li class="sidebar-title">
-      <span>Active endpoint</span>
-    </li>
+    <li class="sidebar-title"><span>Active endpoint</span></li>
     <li class="sidebar-title">
       <select class="select-endpoint form-control" ng-options="endpoint.Name for endpoint in endpoints" ng-model="activeEndpoint" ng-change="switchEndpoint(activeEndpoint)">
       </select>
     </li>
     <li class="sidebar-title"><span>Endpoint actions</span></li>
     <li class="sidebar-list">
-      <a ui-sref="dashboard" ui-sref-active="active">Dashboard <span class="menu-icon fa fa-tachometer"></span></a>
+      <a ui-sref="dashboard" ui-sref-active="active">Dashboard <span class="menu-icon fa fa-tachometer fa-fw"></span></a>
```

Comment:
Quick question, why is the sidebar affected by this PR?

Reply:
Some of the intermediate tests we did involved adding an item to the sidebar. When a first level item was added with the "fa-heart" icon, this was not properly aligned with the others, because of the width. `fa-fw` is specially meant to make all the icons in a list have the same width: http://fontawesome.io/examples/

Although we now removed the heart icon, because we added "About" as a second level item, I think that it is good to keep `fa-fw`. I can split it to a different PR if you prefer.